hoodie-common/src/main/java/com/uber/hoodie/common/util/ReflectionUtils.java

```diff
@@ -58,4 +46,28 @@
         }
     }
 
+    /**
+     * Instantiate a given class with a generic record payload
+     *
+     * @param recordPayloadClass
+     * @param payloadArgs
+     * @param <T>
+     * @return
+     */
+    public static <T extends HoodieRecordPayload> T loadPayload(String recordPayloadClass,
+                                                                       Object [] payloadArgs,
```

Comment:
indentation.

Reply:
corrected. My intellij keeps changing the indentation to 4 spaces instead of 2. Fixed it now.